jujugui/static/gui/src/app/app.js

```diff
@@ -2904,6 +2849,7 @@ YUI.add('juju-gui', function(Y) {
       this._renderZoom();
       this._renderBreadcrumb();
       this._renderHeaderSearch();
+      this._renderModals();
```

Comment:
`_renderComponents` gets called every time the db changes (often) do we really need to render the settings modals every time?